lib/model/rwfolder.go

```diff
@@ -1345,7 +1291,12 @@ func (f *sendReceiveFolder) copierRoutine(in <-chan copyBlocksState, pullChan ch
 				state.copyDone(block)
 			}
 		}
-		weakHashFinder.Close()
```

Comment:
We process this in a loop, so we can't defer file closing, unless we move all of the body of the loop to a function.